src/features/relatedNotes/index.tsx

```
@@ -1,11 +1,11 @@
-import { PluginSettings } from "@/constants/settings";
-import { AppContext, SettingsContext } from "@/helpers/context";
+import { PLUGIN_APP_ID } from "@/shared/constants";
+import { PluginSettings } from "@/shared/constants/settings";
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)